src/climate_indices/compute.py

```diff
@@ -743,3 +743,267 @@ def transform_fitted_gamma(
+        probabilities = scipy.stats.fisk.cdf(values, c, scale, loc)
+
+        if not np.all(np.isnan(probabilities)):
+
+            # the values we'll return are the values at which the probabilities
+            # of a normal distribution are less than or equal to the computed
+            # probabilities, as determined by the normal distribution's
+            # quantile (or inverse cumulative distribution) function
+            fitted_values = scipy.stats.norm.ppf(probabilities)
+
+        else:
+
+            fitted_values = values
+
+    else:
+
+        fitted_values = values
+
+    return fitted_values
+
+
+# ------------------------------------------------------------------------------
+# TODO uncommenting numba.jit leads to problems here...
+# @numba.jit
+def transform_fitted_fisk(
+        values: np.ndarray,
+        data_start_year: int,
+        calibration_start_year: int,
+        calibration_end_year: int,
+        periodicity: Periodicity,
+        probabilities_of_zero: np.ndarray = None,
+        locs: np.ndarray = None,
+        scales: np.ndarray = None,
+        c: np.ndarray = None,
+) -> np.ndarray:
+    """
+    Fit values to a log-logistic/fisk distribution and transform the values
+    to corresponding normalized sigmas.
+
+    :param values: 2-D array of values, with each row representing a year containing
+                   twelve columns representing the respective calendar months,
+                   or 366 columns representing days as if all years were leap years
+    :param data_start_year: the initial year of the input values array
+    :param calibration_start_year: the initial year to use for the calibration period
+    :param calibration_end_year: the final year to use for the calibration period
+    :param periodicity: the periodicity of the time series represented by the input
+                        data, valid/supported values are 'monthly' and 'daily'
+                        'monthly' indicates an array of monthly values, assumed
+                        to span full years, i.e. the first value corresponds
+                        to January of the initial year and any missing final
+                        months of the final year filled with NaN values,
+                        with size == # of years * 12
+                        'daily' indicates an array of full years of daily values
+                        with 366 days per year, as if each year were a leap year
+                        and any missing final months of the final year filled
+                        with NaN values, with array size == (# years * 366)
+    :param probabilities_of_zero: pre-computed probabilities of zero for each
+        month or day of the year
+    :param locs: pre-computed loc values for each month or day of the year
+    :param scales: pre-computed scale values for each month or day of the year
+    :param skews: pre-computed skew values for each month or day of the year
```

Comment:
No `skews` argument, and missing documentation for `c`. Also, I am not a fan of single letter variable names. Is this `c` parameter known by any other names?

Reply:
Agreed, meaningful variable names are preferred.